### PR TITLE
Add display_name

### PR DESCRIPTION
**Pull request by nanduni-nin**
Add missing `display_name` attribute to all manifests.

Resolves: #4

### CODE REVIEW

**Comment by nanduni-nin** [comment]
I added `display_name` to all libraries except following.
1. `echonest` : Not sure if it is `libechonest` or `echonest`.
2. `kdtools` : Current home page does not exist.
3. `libmm-qt` : Invalid project.

File: kdelibs/kdelibs.2013-09-03.manifest [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "http://inqlude.org/schema/release-manifest-v1#",
   "name": "kdelibs",
+  "display_name": "KDE TechBase",
```
Comment (cornelius) [inline kdelibs/kdelibs.2013-09-03.manifest]: This should be "KDE 4 Libraries".

File: kdepimlibs/kdepimlibs.2013-09-03.manifest [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "http://inqlude.org/schema/release-manifest-v1#",
   "name": "kdepimlibs",
+  "display_name": "KDE PIM",
```
Comment (cornelius) [inline kdepimlibs/kdepimlibs.2013-09-03.manifest]: This should be "KDE PIM Libraries".

File: o2/o2.2012-06-09.manifest [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "http://inqlude.org/schema/release-manifest-v1#",
   "name": "o2",
+  "display_name": "OAuth 1.0 and 2.0 for Qt",
```
Comment (cornelius) [inline o2/o2.2012-06-09.manifest]: A one-word display name would be better.

File: tasks/tasks.2015-12-26.manifest [hunk 4]
```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "http://inqlude.org/schema/release-manifest-v1#",
   "name": "tasks",
+  "display_name": "tasks",
```
Comment (cornelius) [inline tasks/tasks.2015-12-26.manifest]: Is this the upstream name? A properly capitalized name would be better.

**Pull request closed without merging**